### DISCUSSION TITLE
1.5

### DISCUSSION THREAD

**Discussion by adellanuki** [hlky/stable-diffusion#291] [Q&A] [answered]
Because I'm ignorant and don't fully understand, is 1.5 already added in API?(whatever that means)
![image](https://user-images.githubusercontent.com/94848873/188533729-efc41ea2-632c-4165-bd92-bef5d87eaf8c.png)

I'd be lying if I said I knew how it worked.

**Comment by Kyxsune** [comment, marked as answer]
The API they are referring to is the proprietary sdk - the service for SD put forward by Stability AI. Using this SDK is not free and has some of the same restrictions as dreamstudio.

It also, is not publicly available like v1.4 at this time, but they intend to release it to the public later.

**Reply by adellanuki** [reply]
Thank you for clearing that up.
